scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1056,6 +1255,9 @@ $DirectMethodModuleToModuleDeploymentFilename = "dm_module_to_module_deployment.
 $RuntimeOnlyDeploymentFilename = 'runtime_only_deployment.template.json'
 $QuickstartDeploymentFilename = 'quickstart_deployment.template.json'
 $TwinTestFilename = "twin_test_tempSensor.json"
+# Using versions without snitcher and influxdb, as they are currently not working in windows
+$LongHaulDeploymentFilename = "long_haul_deployment_windows.template.json"
```

Comment:
Define functions to return long haul and stress deployment file path.

Reply:
Why does this need a function when the other deployment files above don't?

Reply:
The path of long haul and stress deployment file should be different bewtween x64 and arm32.  Therefore it is better to have a function for it.